lib/python-sdk/examples/list_opportuntites.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""Example script demonstrating basic client usage.
+
+Run with: poetry run python list_opportuntites.py
+"""
+
+from common_grants_sdk.client import Client
+from common_grants_sdk.client.config import Config
+
+config = Config(
+    base_url="http://localhost:8080",
+    api_key="two_orgs_user_key",
+    timeout=5.0,
+    page_size=10,
+)
```

Comment:
It could make sense to have a default page size configured on client creation, but SDK users should be able to specify page size per request as well.